tests/tag/core/test_systemtimestamper.py

```diff
@@ -17,6 +18,7 @@ def test__after_delay__returns_current_time(self):
         time.sleep(0.05)
         _assert_time_within_tolerance(_gmt_now(), uut.timestamp)
 
+    @pytest.mark.skip(reason="This test isn't running fast enough in GitHub Actions")
```

Comment:
The runner for GitHub actions doesn't seem to be fast enough for this test to do what it's supposed to. I'm looking into using a larger runner for this repo, but for now I just disabled the test.

Reply:
You mentioned you had reached out to some GitHub admins related to this? Do we have any updates on that?